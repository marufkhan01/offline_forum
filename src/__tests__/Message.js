import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Message from '../components/Bot/Message';

it('Button component should render as expected',() =>{
    const component = shallow(<Message />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
});


