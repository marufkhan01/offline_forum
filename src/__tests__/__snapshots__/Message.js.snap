// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component should render as expected 1`] = `
<p
  className="bg-indigo-dark text-white font-bold py-2 px-4 mb-2 rounded-full h-8"
/>
`;
